### PR TITLE
Malicious Infrastructure: pt-br translation of Module 1

### PR DESCRIPTION
**Pull request by yamakat**

### CODE REVIEW

**Review by yamakat — COMMENTED** [review]
Updating with minor adjustments in grammar.

File: content/learning-path/1/module-1.pt-br.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+⚠️ Ao considerar ou conduzir de fato uma investigação, garanta um equilíbrio entre redução de danos e suporte de quaisquer alvos, como aquelas listadas na [Recuperação de possível conta comprometida (securityinabox.org)](https://securityinabox.org/pt/communication/account-compromise/).
+
+## Checagem de habilidades
+
+Dedique algum tempo ao quiz do [Shira (em inglês)](https://shira.app/) até que se sinta apte a passar nos testes confortavelmente, e reconhecer phishing em diversas categorias de apps.
+
+## Recursos de aprendizagem
+
+{{% resource title="Shira, por Horizontal" languages="Inglês, Espanhol, Mandarin" cost="Grátis" description="Quiz online com amostras de e-mails que usuáries devem determinar se são maliciosos ou não" url="https://shira.app" %}}
```
[the remark this answers is not in the document]
Comment (contraexemplo) [reply content/learning-path/1/module-1.pt-br.md:49]:
```suggestion
{{% resource title="Shira, por Horizontal" languages="Inglês, espanhol, mandarim" cost="Grátis" description="Quiz online com amostras de e-mails que usuáries devem determinar se são maliciosos ou não" url="https://shira.app/" %}}
```

File: content/learning-path/1/module-1.pt-br.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+{{% resource title="6 ataques comuns de phishing e como se proteger contra eles" languages="Inglês" cost="Grátis" description="Um resumo de alguns dos ataques comuns de phishing, incluindo métodos mais sofisticados" url="https://www.tripwire.com/state-of-security/6-common-phishing-attacks-and-how-to-protect-against-them" %}}
+
+{{% resource title="5 técnicas comuns de phishing" languages="Inglês" cost="Grátis" description="Perspectiva sobre algumas técnicas utilizadas por atacantes para tornar os e-mails mais convincentes e indetectáveis" url="https://www.vadesecure.com/en/blog/5-common-phishing-techniques" %}}
+
+{{% resource title="CiviCERT" languages="Inglês" cost="Grátis" description="Rede de organizações da sociedade civil e grupos de resposta rápida que se concentram em ataques cibernéticos e similares" url="https://www.civicert.org/" %}}
+
+{{% resource title="Kit de primeiros socorros digitais" languages="Árabe, Espanhol, Farsi, Francês, Indonésio, Armênio, Quirguistanês, Burmês, Português, Russo, Albanês, Tailandês, Ucraniano" cost="Grátis" description="Guia de apoio a protetores digitais que lidam com uma variedade de problemas" url="https://digitalfirstaid.org/pt/" %}}
```
[the remark this answers is not in the document]
Comment (contraexemplo) [reply content/learning-path/1/module-1.pt-br.md:59]:
```suggestion
{{% resource title="Kit de primeiros socorros digitais" languages="Árabe, espanhol, farsi, francês, indonésio, armênio, quirguistanês, burmês, português, russo, albanês, tailandês, ucraniano" cost="Grátis" description="Guia de apoio a protetores digitais que lidam com uma variedade de problemas" url="https://digitalfirstaid.org/pt/" %}}
```

File: content/learning-path/1/module-1.pt-br.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+{{% resource title="Recuperação de possível conta comprometida" languages="Árabe, Indonésio, Inglês, Espanhol, Farsi, Francês, Português, Russo, Tailandês, Turco, Vietnamita, Chinês, Tibetano, Quemer, Burmês" cost="Grátis" description="Guia com passos imediatos e a longo prazo no comprometimento de contas digitais" url="https://securityinabox.org/pt/communication/account-compromise/" %}}
```
[the remark this answers is not in the document]
Comment (contraexemplo) [reply content/learning-path/1/module-1.pt-br.md:61]:
```suggestion
{{% resource title="Recuperação de possível conta comprometida" languages="Árabe, indonésio, inglês, espanhol, farsi, francês, português, russo, tailandês, turco, vietnamita, chinês, tibetano, quemer, burmês" cost="Grátis" description="Guia com passos imediatos e a longo prazo no comprometimento de contas digitais" url="https://securityinabox.org/pt/communication/account-compromise/" %}}
```

**Pull request merged**